src/components/Markers.jsx

```diff
@@ -31,7 +31,7 @@ export const Markers = ({
   const renderMarker = (event) => {
     const { coords, title, date, source, id, tipoId, tipo } = event;
     // Document this: Where does 'hovered' come from? It's not in the stylesheet
```

Comment:
this comment should probably go away.

Reply:
Good catch + since I was the one who made the comment I'll make sure to link an issue next time I leave a TODO or FIXME in the future